universum/configuration_support.py

```diff
@@ -75,6 +75,11 @@ class Step:
         as a separate artifact, so when using ``--no-archive`` option, do not claim directories as `report_artifacts`.
         Please note that any file can be put as `artifact`, `report_artifact`, or both. A file that is both
         in `artifacts` and `report_artifacts`, will be mentioned in a report and will cause build failure when missing.
+
+        .. note::
+
+            GitHub Actions doesn't support this key.
```

Comment:
As far as I understand the artifacts key also is not supported by GitHub Actions.
Is that correct?

Reply:
It is not correct. The `artifacts` key copies files to the artifacts folder, and GA can store artifacts from there.

Reply:
Just a note to myself: regular artifacts are not reported in the build log and for them the GitHub Action implementation is not called.